### PR TITLE
added conditional class .active to Nav component

### PR DESCRIPTION
**Pull request by JoelLau**
Resolves Issue #73 

- added class `.active` to active route
- conditionally applied class on `nav-link`s
    - [using sapper segments](https://sapper.svelte.dev/docs#Nested_routes)

### CODE REVIEW

**Comment by humancalico** [comment]
Looks good. Thanks @JoelLau. Can you run `prettier` otherwise it seems fine to me

File: src/components/Nav.svelte [hunk 1]
```diff
@@ -190,6 +192,8 @@ li:not(:last-child) {
 </style>
 
 <script>
+export let segment;
+console.log(segment);
```
Comment (humancalico) [inline src/components/Nav.svelte:196]: @JoelLau just one more thing. Is this still needed or was this just for testting purpose?

Comment (JoelLau) [reply src/components/Nav.svelte]: woops. good catch! I left it in there for testing reasons. it has been removed

**Review by aayushmau5 — APPROVED** [review]

**Comment by aayushmau5** [comment]
Awesome work. Thanks @JoelLau

**Pull request merged**